extensions/linz/schema.json

```diff
@@ -1047,16 +1075,12 @@
               "required": ["minimum", "maximum"],
               "properties": {
                 "minimum": {
-                  "title": "Earliest asset creation time",
-                  "type": "string",
-                  "format": "date-time",
-                  "pattern": "(\\+00:00|Z)$"
+                  "title": "Earliest metadata creation time",
```

Comment:
I know that the required fields for this are removed in [https://github.com/linz/stac/pull/187](https://github.com/linz/stac/pull/187), which is good.
I am not sure what the source of these summaries fields would be, as I don't think (but please correct me if wrong!) there is a corresponding field at the Item level (i.e. for the metadata), only at the Asset level.

Reply:
This is the actual creation and update time of the STAC metadata file itself instead of the creation and update time of the Asset file.

Reply:
There will be scenarios where the metadata is updated independently of the assets being updated e.g. STAC specification changes or new metadata that we have standardised and can now add...

Reply:
Ahh thanks it's in the Common Metadata! [https://github.com/radiantearth/stac-spec/blob/master/item-spec/common-metadata.md](https://github.com/radiantearth/stac-spec/blob/master/item-spec/common-metadata.md)
item.json -> datetime.json